spec/requests/api/subscriber/can_rate_the_article_spec.rb

```diff
@@ -34,7 +34,7 @@
     end
 
     it 'is expected that member have a single rating' do
```

Comment:
should it say subscriber and not member here? :)

Reply:
Good spot will fix it with the next PR.

Reply:
👍